src/router.js

```diff
@@ -65,13 +65,15 @@ export const routerFactory = () => {
     if (to.matched.some(record => record.meta.requiresAuth)) {
       // page only visible when logged in
 
+      // TODO: remove before merging
+      return next()
```

Comment:
Hey @robbiecarlton 
Just tagging along for the ride here :p anyways I noticed this got left in and says it should be removed before merge... could cause issues? Worth a second look I guess. 

Cheers,
Connor

Reply:
Thanks @Connoropolous, good catch!